src/build/parse_xml_file.mjs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 The Outline Authors
```

Comment:
It can be in another PR, but this should be in `src/infrastructure` instead. It's not really build-specific. (also build should mostly move there too)